src/components/AccountCreation.tsx/AccountCreation.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Very small nitpick, but please put these disabling comments at the very top of the file:

```suggestion
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
import React, { useState } from "react";

```

Reply:
Sorry I was wrong, we should be using `@ts-nocheck` instead of `@ts-ignore`!